mirte_robot/robot.py

```diff
@@ -260,18 +528,25 @@ def setOLEDText(self, oled, text):
             oled (str): The name of the sensor as defined in the configuration.
             text (str): String to be shown on the 128x64 OLED.
         """
-        value = self.oled_services[oled]('text', str(text))
+        value = self._call_service(
+            self.oled_services[oled]["text"], SetOLEDText.Request(text=str(text))
+        )
         return value.status
 
     def setOLEDImage(self, oled, image):
         """Shows image on the OLED.
 
         Parameters:
             oled (str): The name of the sensor as defined in the configuration.
-            image (str): Image name as defined in the images folder of the mirte-oled-images repository (excl file extension).
+            image (str): Image name as defined in the images folder of the mirte-oled-images repository (optionally excl file extension).
```

Comment:
This can also be an absolute path to an image?

Reply:
and ros pkg path

Reply:
@ArendJan, What do you think about the updated documentation

Reply:
not great, but sure for now, I'll create an issue for this to just get this PR merged